Documentation/ApiOverview/Mail/Index.rst

```diff
@@ -117,6 +162,51 @@ mbox
 How to Create and Send Mails
 ============================
 
+Both :php:`\TYPO3\CMS\Core\Mail\MailMessage` and :php:`\TYPO3\CMS\Core\Mail\FluidEmail) inherit
```

Comment:
There is a closing `)` instead of single back tick, that breaks the rendered result.